app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    new_comment = Comment.new(author: current_user, post:, text: comment_params)
```

Comment:
Thank you for the review @Dipeshtwis. The code is this way because rubocop is forcing me to remove the hash value for `post`. I am guessing it is because of the `post` variable already on line 3